deploy/clowder.md

```diff
@@ -3,47 +3,48 @@
 
 * Install `bonfire` following the instructions [here](https://github.com/RedHatInsights/bonfire#installation)
 
-* Configure `bonfire` to use your checkout.  This cat command is just a
-  short-cut so the instructions will be succinct.  You should open the file and
-  paste in the name and component bits yourself under the `apps:` key.  If you
-  paste in the contents, replace `$(pwd)` with the directory where your
-  subscription-watch checkout is
-
-  You can override parameters as shown below, or alternatively with the bonfire
-  `-p` argument during the deploy step.  The parameters in the example below are
-  useful for development environments.
-
+* Configure `bonfire` to use your checkout.  Use bonfire's write-default command, then edit ~/.config/bonfire/config.yaml
   ```
   bonfire config write-default
-
-  cat <<BONFIRE >>  ~/.config/bonfire/config.yaml
+  ```
+  You can override parameters by editing the config file, or alternatively with the bonfire `-p` argument during the deployment step.  The configurations in the example below are useful for development environments.  Be sure to update this example with the correct path to where your rhsm-subscription repo is located.  
```

Comment:
Why not leave the bash in? Should be able to just replace your path with `$(pwd)` throughout.

Reply:
Because the yaml in the bash was out of date (had capacity-allowlist and swatch-producer-aws as separate components).  All of our other ClowdApp templates have their own README files with an example yaml config.  was trying to eliminate confusion and/or dead once duplicated documentation